src/aircraft/FenixA320.ts

```diff
@@ -87,15 +113,39 @@ export default class FenixA320 extends AircraftConfig {
     return value === 2
   }
 
+  taxiLights(value: number): FeatureState {
+    return value === 1 || value === 2
+  }
+
   wingLights(value: number): FeatureState {
     return value === 1
   }
 
-  taxiLights(value: number): FeatureState {
+  emergencyLights(value: number): FeatureState {
     return value === 1 || value === 2
   }
 
-  APU(mastersw: number, startsw: number): FeatureState {
-    return mastersw === 1 && startsw === 2
+  seatbelts(value: number): FeatureState {
+    return value === 1
+  }
+
+  battery(b1: number, b2: number, ext_pwr: number): FeatureState {
+    return (b1 === 1 && b2 === 1) || ext_pwr === 1
+  }
+
+  APU(master_switch: number, avail_status: number): FeatureState {
```

Comment:
This should be `apu` - it's case sensitive

Reply:
Fixed